marklogic-data-hub-central/ui/src/components/entities/mapping/create-edit-mapping-dialog/create-edit-mapping-dialog.test.tsx

```diff
@@ -89,7 +89,15 @@ describe('Create/Edit Mapping Step artifact component', () => {
     const queryInput = getByPlaceholderText('Enter Source Query');
     fireEvent.change(queryInput, { target: {value: 'cts.collectionQuery(["testCollection"])'}});
     expect(queryInput).toHaveTextContent('cts.collectionQuery(["testCollection"])');
-
+    fireEvent.change(queryInput, { target: {value: 'cts.collectionQuery("testCollection")'}});
+    expect(queryInput).toHaveTextContent('cts.collectionQuery("testCollection")');
+    expect(collInput).toHaveValue("testCollection");
+    fireEvent.change(queryInput, { target: {value: "cts.collectionQuery(['testCollection'])"}});
+    expect(queryInput).toHaveTextContent("cts.collectionQuery(['testCollection'])");
+    expect(collInput).toHaveValue('testCollection');
+    fireEvent.change(queryInput, { target: {value: "cts.collectionQuery('testCollection')"}});
+    expect(queryInput).toHaveTextContent("cts.collectionQuery('testCollection')");
+    expect(collInput).toHaveValue('testCollection');
```

Comment:
I will file a separate bug for the following issue. If you change the query to "test", collInput will still show up as "testCollection" ie change in sourceQuery is not reflected in sourceCollection